esm_master/task.py

```diff
@@ -354,6 +359,18 @@ def assemble_command_list(self):
                                 + "/"
                                 + binfile.split("/", -1)[-1]
                             )
+
+        import subprocess
+        import sys
+
+        def install(package):
+            subprocess.check_call([sys.executable, "-m", "pip", "install", "--user", package])
```

Comment:
Should these lines be moved somewhere else?

Reply:
Probably should go at the top, yes